Mathlib/NumberTheory/AbelSummation.lean

```diff
@@ -83,19 +83,20 @@ private theorem ineqofmemIco' {k : ℕ} (hk : k ∈ Ico (⌊a⌋₊ + 1) ⌊b⌋
     a ≤ k ∧ k + 1 ≤ b :=
   ineqofmemIco (by rwa [← Finset.coe_Ico])
 
-private theorem integrablemulsum (ha : 0 ≤ a) {g : ℝ → 𝕜} (hg_int : IntegrableOn g (Set.Icc a b)) :
-    IntegrableOn (fun t ↦ g t * ∑ k ∈ Icc 0 ⌊t⌋₊, c k) (Set.Icc a b) := by
+theorem _root_.integrableOn_mul_sum {m : ℕ} (ha : 0 ≤ a) {g : ℝ → 𝕜}
```

Comment:
If the theorem is to be "de-privatized", then probably it should get a slightly more specific name (`integrableOn_mul_sum_Icc_floor` perhaps?) so people don't find it by accident when searching for general analysis lemmas about integrals and sums.

Reply:
I went with the slightly shorter `integrableOn_mul_sum_Icc`. I hope that's fine.